README.md

```diff
@@ -187,24 +163,6 @@ docker-compose run --rm web bin/rake argo:repo:load
 
 ## Common tasks
 
-### Run the tests
```

Comment:
@ndushay you OK with this change? I can restore if not. Didn't make sense to me to have testing covered in two spots.

Reply:
totally fine with me.